app/scripts/migrations/120.1.ts

```diff
@@ -0,0 +1,37 @@
+import { cloneDeep } from 'lodash';
+import { hasProperty } from '@metamask/utils';
+import { InternalAccount } from '@metamask/keyring-api';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 120.1;
+
+/**
+ * Add a default value for importTime in the InternalAccount
+ *
+ * @param originalVersionedData
+ */
+export async function migrate(
+  originalVersionedData: VersionedData,
+): Promise<VersionedData> {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  transformState(versionedData.data);
+  return versionedData;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We should use `unknown` here rather than `any`. See here for why we should be avoiding the use of `any` in general: https://github.com/MetaMask/contributor-docs/blob/main/docs/typescript.md#avoid-any

Reply:
Valid, this was copied from a previous migration. Migrated to using `unknown`